test/plugins/test_linux_log_parser.py

```diff
@@ -259,7 +259,7 @@ def test_sha_name(self):
         self.assertNotIn('Kernel panic', test.log)
 
         # Now check if a test with sha digest in the name
-        test = testrun.tests.get(suite__slug='log-parser-boot', metadata__name='check-kernel-oops-a1acf2f0467782c9c2f6aeadb1d1d3cec136642b13d7231824a66ef63ee62220')
+        test = testrun.tests.get(suite__slug='log-parser-boot', metadata__name='check-kernel-oops-oops-bug-preempt-smp-a1acf2f0467782c9c2f6aeadb1d1d3cec136642b13d7231824a66ef63ee62220')
```

Comment:
We could do this. This would mean the tests created would look like:
- check-kernel-oops-bug-preempt-smp
- check-kernel-oops-bug-preempt-smp-<sha>
- check-kernel-oops-bug-preempt-smp-<sha>
- ...etc
- check-kernel-<name2>
- check-kernel-<name2>-<sha>
- check-kernel-<name2>-<sha>
- ...etc

Instead of:
- check-kernel-oops
- check-kernel-oops-bug-preempt-smp-<sha>
- check-kernel-oops-bug-preempt-smp-<sha>
- ...etc
- check-kernel-<name2>-<sha>
- check-kernel-<name2>-<sha>
- ...etc

So there would be more tests generated but each "big bucket" (before dividing by sha) would have less in it.

Is change helpful?

Reply:
yes, make sense and its super helpful =)

Reply:
I have pushed an update to move the extracted name regex to the higher-level log parser bucket as discussed here.

The name is still like "check-kernel-oops-oops-bug-preempt-smp" rather than "check-kernel-oops-bug-preempt-smp" since I didn't want to risk losing the "oops" part (or equivalent) in case the naming regex fails to extract anything (so we can fall back on "check-kernel-oops" in that case).

Let me know what you think :)